cmd/root.go

```diff
@@ -31,7 +31,9 @@ var (
 
 		$ stellar auth activate-api-key path/to/stellarstation-private-key.json
 
-		All commands should work after that.`)
+		All commands should work after that.
+
+		Notice: The open stream flag '--enable-auto-close' which is supplied with an '--auto-close-time' has been deprecated and will be removed in a future release.'`)
```

Comment:
This should be in the help section next to the actual option description. Not in the general help of the tool.

Reply:
Removed this.